gst/nnstreamer/include/meson.build

```diff
@@ -0,0 +1,25 @@
+# Common headers to be installed
+nnst_common_headers = [
+  'tensor_typedef.h',
+  'tensor_filter_custom.h',
+  'tensor_filter_custom_easy.h',
+  'nnstreamer_plugin_api_filter.h',
+  'nnstreamer_plugin_api_decoder.h',
+  'nnstreamer_plugin_api_converter.h',
+  'nnstreamer_plugin_api.h'
+]
+
+foreach h : nnst_common_headers
+  nnstreamer_headers += join_paths(meson.current_source_dir(), h)
+endforeach
+
+# Install headers into /{includedir}/nnstreamer
+install_headers(nnstreamer_headers,
+  subdir: 'nnstreamer'
+)
+
+if get_option('enable-filter-cpp-class')
```

Comment:
Question: tensor_filter_cpp.hh
is it also required to move include dir and install if enable-cppfilter is true?

Reply:
Yes. it will be installed to include dir.